<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stegano Explorer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>TOOLS</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="newFolderPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Create new folder</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/assets/img/folder.png</normaloff>:/assets/img/folder.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="newFilePushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Create new file</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/assets/img/document.png</normaloff>:/assets/img/document.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="deletePushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Delete Selection</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/assets/img/delete(1).png</normaloff>:/assets/img/delete(1).png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cutPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Cut selection</string>
             </property>
             <property name="whatsThis">
              <string>cut selected files</string>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/assets/img/scissors(1).png</normaloff>:/assets/img/scissors(1).png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="copyPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Copy</string>
             </property>
             <property name="whatsThis">
              <string>cut selected files</string>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/assets/img/copy.png</normaloff>:/assets/img/copy.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pastePushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Paste</string>
             </property>
             <property name="whatsThis">
              <string>cut selected files</string>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/assets/img/paste.png</normaloff>:/assets/img/paste.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="previewToolButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Show preview</string>
             </property>
             <property name="whatsThis">
              <string>show preview</string>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Neuer Ordner</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/assets/img/preview (2).png</normaloff>:/assets/img/preview (2).png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="darkModePushButton">
             <property name="toolTip">
              <string>Light Mode</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/assets/img/light-bulb.png</normaloff>:/assets/img/light-bulb.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/assets/img/arrow-back.png</normaloff>:/assets/img/arrow-back.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="forwardButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/assets/img/arrow-forth.png</normaloff>:/assets/img/arrow-forth.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pathLineEdit">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchLineEdit">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="DisplayComboBox">
          <item>
           <property name="text">
            <string>Table</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Icons</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sortComboBox">
          <property name="currentText">
           <string>Alphabetically</string>
          </property>
          <item>
           <property name="text">
            <string>Alphabetically</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Size</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Date</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionNewFolder"/>
    <addaction name="separator"/>
    <addaction name="actionMount"/>
    <addaction name="actionUnmount"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Filesystem_information"/>
    <addaction name="actionDefragment"/>
    <addaction name="actionFormat_Filesystem"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVorschau_anzeigen">
   <property name="text">
    <string>Vorschau anzeigen</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>New File...</string>
   </property>
  </action>
  <action name="actionNewFolder">
   <property name="text">
    <string>New Folder...</string>
   </property>
  </action>
  <action name="actionDefragment">
   <property name="text">
    <string>Defragment</string>
   </property>
  </action>
  <action name="actionCreate_new">
   <property name="text">
    <string>Create New</string>
   </property>
  </action>
  <action name="actionShow_Filesystem_information">
   <property name="text">
    <string>Show Filesystem information</string>
   </property>
  </action>
  <action name="actionLoad_selected_file">
   <property name="text">
    <string>Load selected file</string>
   </property>
  </action>
  <action name="actionChoose_from_explorer">
   <property name="text">
    <string>Choose from explorer</string>
   </property>
  </action>
  <action name="actionUnload">
   <property name="text">
    <string>Unload</string>
   </property>
  </action>
  <action name="actionMount">
   <property name="text">
    <string>Mount</string>
   </property>
  </action>
  <action name="actionUnmount">
   <property name="text">
    <string>Unmount</string>
   </property>
  </action>
  <action name="actionFormat_Filesystem">
   <property name="text">
    <string>Format Filesystem</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
